utils.groovy

```diff
@@ -820,4 +820,92 @@ def matrixSend(message) {
     }
 }
 
+def build_remote_image(arches, commit, url, repo, tag, yumrepo=None, from=None,
+                        extraArgs=None) {
+     // Note we can pass ARCH_REPLACE keyword in the tag,
+     // in this way we make the process easier to handle arch tag names.
+    withCredentials([file(credentialsId: 'oscontainer-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
```

Comment:
We shouldn't hardcode the secret name here. I think probably best is to leave that to the calling job, which can pass in the registry file path as another parameter?